cms/djangoapps/contentstore/views/course.py

```diff
@@ -810,6 +810,16 @@ def _create_or_rerun_course(request):
                     status=400
                 )
 
+        # Allow user to create the course only if they belong to the organisation
+        # This flag doesn't apply to Global Staff and Superusers
+        if settings.FEATURES.get('RESTRICT_COURSE_CREATION_TO_ORG_ROLES', False):
+            has_org_permission = has_studio_write_access(request.user, None, org)
+            if not has_org_permission:
+                return JsonResponse(
```

Comment:
@farhaanbukhsh, it will be a good idea to add a log line incorporating the suggestion [here](https://github.com/open-craft/edx-platform/pull/319/files#r580507182).

Reply:
done